author/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+class Author(models.Model):
+    type =  models.CharField(max_length=255)
+
+    # ID of the Author
+    id=	models.CharField(max_length=255, primary_key=True)
```

Comment:
why? id field is automatically created by a model. maybe we need a uuid for the url slugs, but id shouldnt be needed